samples/openthread/cli/snippets/usb/usb.overlay

```diff
@@ -6,12 +6,12 @@
 
 / {
 	chosen {
-		zephyr,ot-uart = &cdc_acm_uart;
+		zephyr,shell-uart = &snippet_cdc_acm_uart;
```

Comment:
```suggestion
		zephyr,ot-uart = &snippet_cdc_acm_uart;
```
this should remain unchanged since this is a name referenced in:
https://github.com/nrfconnect/sdk-zephyr/blob/main/modules/openthread/platform/uart.c#L43